ConstructorGenerator.Tests/TestClasses/TestClassWithBaseClassAndFullConstructorAttribute.cs

```diff
@@ -0,0 +1,31 @@
+using ConstructorGenerator.Attributes;
+
+namespace ConstructorGenerator.Tests.TestClasses;
+
+[GenerateFullConstructor]
+public partial class TestClassWithBaseClassAndFullConstructorAttribute : BaseClassWithFullConstructorAttribute
+{
+    public readonly string FieldWithInitializer = null!;
+    [ConstructorDependency] public readonly string FieldWithInitializerButAttribute = null!;
+    
+    public string PropertyWithInitializer { get; } = null!;
+    [ConstructorDependency] public string PropertyWithInitializerButAttribute { get; } = null!;
+
+    public string PropertyWithInit { get; init; }
+
+    public string PropertyWithSetter { get; set; }
+    [ConstructorDependency]
+    public string PropertyWithSetterButAttribute { get; set; }
+    
+    
+    public string ComputedProperty => null!;
+
+    public readonly string ReadonlyDependency;
+    
+    public string PropertyReadOnly { get; }
+    
+    [ExcludeConstructorDependency] public readonly string ReadonlyDependencyButIgnored = null!;
```

Comment:
I'm not sure if this is the best test. That members would be excluded anyway because they have an initializer, right?